terraform/modules/bigquery_metrics_views/data/bq_tvf/resources/pull_requests_by_date.sql

```diff
@@ -80,11 +80,11 @@ FROM
         INNER JOIN (
         SELECT
             id,
-            MAX(received) received
+            MAX_BY(delivery_id, updated_at) delivery_id
```

Comment:
Ideally, it would be MAX_BY(deliveryid,  updated_at, received) as updated_at only has second granularity, so received would be able to break ties.

Also would be nice if a null updated_at was handled gracefully, but I don't think we do that upstream anyways, so its not really an issue.

Reply:
`update_at` is listed as required in the GitHub docs. So I think it should never be null. https://docs.github.com/en/webhooks/webhook-events-and-payloads#pull_request

Checking against our current data set, we don't have any pull request events with a null `updated_at`